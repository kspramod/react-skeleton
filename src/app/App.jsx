import Dashboard from './containers/Dashboard';

import logo from '../assets/logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        Sample React Skeleton
      </header>
      <Dashboard />
    </div>
  );
}

export default App;
