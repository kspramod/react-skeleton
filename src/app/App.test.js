import { render, screen } from '@testing-library/react';

import App from './App';

describe('<App />', () => {
  test('Snapshot test', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });

  test('renders Sample React Skeleton `header`', () => {
    render(<App />);
    const pageHeader = screen.getByText(/Sample React Skeleton/i);
    expect(pageHeader).toBeInTheDocument();
  });
});
