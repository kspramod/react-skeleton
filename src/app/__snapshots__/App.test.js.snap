// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Snapshot test 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <img
        alt="logo"
        class="App-logo"
        src="logo.svg"
      />
      Sample React Skeleton
    </header>
    <div>
      <h1>
        Dashboard
      </h1>
    </div>
  </div>
</div>
`;
